config.js

```diff
@@ -0,0 +1,170 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
```

Comment:
This should stay inside the tokens package - it's a style-dictionary config that should only be leveraged from there.

Reply:
Done